package/devel/tmon/Makefile

```diff
@@ -0,0 +1,58 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=tmon
+PKG_VERSION:=$(shell basename $(LINUX_DIR) | sed 's/linux-//')
```

Comment:
perf and spidev_test are using:
````
PKG_RELEASE:=$(LINUX_VERSION)
````
Why do you do it different here?

Reply:
Thanks for the hint, I'll take another look at that.